io.github.hedge_dev.hedgemodmanager.yml

```diff
@@ -0,0 +1,55 @@
+app-id: io.github.hedge_dev.hedgemodmanager
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  prepend-path: /usr/lib/sdk/dotnet8/bin
+  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
+  prepend-pkg-config-path: /usr/lib/sdk/dotnet8/lib/pkgconfig
+
+command: HedgeModManager.UI
+
+finish-args:
+  - --socket=x11
+  # UnleashedRecomp permissions
+  - --filesystem=xdg-config/UnleashedRecomp:create
+  - --filesystem=xdg-data/applications/UnleashedRecomp.desktop:ro
+  # Create permissions are required for accessing the mod loader, game and for patching game prefixes
+  # Steam permissions
+  - --filesystem=~/.steam/steam/steamapps:create
+  - --filesystem=~/.var/app/com.valvesoftware.Steam/.steam/steam/steamapps:create
+  # Heroic permissions
+  - --filesystem=~/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig:ro
+  - --filesystem=~/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary:ro
+  - --filesystem=~/Games/Heroic:create
+  # Common locations for game and mod files
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=/mnt
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --env=DOTNET_ROOT=/app/lib/dotnet
+
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/dotnet8/bin/install.sh
+  - name: app
```

Comment:
please, don't use generic names.

Reply:
Is a module name like `hedge-mod-manager` fine?

Reply:
it's more descriptive.

Reply:
Updated.